systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -316,6 +332,21 @@ void setupEnvironment() throws InterruptedException {
         kafkaExporterMetricsData = MetricsUtils.collectKafkaExporterPodsMetrics(CLUSTER_NAME);
         kafkaMirrorMaker2MetricsData = MetricsUtils.collectKafkaMirrorMaker2PodsMetrics(MIRROR_MAKER_CLUSTER);
         userOperatorMetricsData = MetricsUtils.collectUserOperatorPodMetrics(CLUSTER_NAME);
+        topicOperatorMetricsData = MetricsUtils.collectTopicOperatorPodMetrics(CLUSTER_NAME);
         clusterOperatorMetricsData = MetricsUtils.collectClusterOperatorPodMetrics();
     }
+    
+    private List<String> getExpectedTopics() {
```

Comment:
Do we rly needs this for `getExpectedTopics().size()` ?

Reply:
I added it because someone can say I am just making up it should be `x`.